docs/Reference/Server.md

```diff
@@ -463,7 +463,7 @@ Defines the label used for the request identifier when logging the request.
 <a id="factory-gen-request-id"></a>
 
 Function for generating the request-id. It will receive the incoming request as
-a parameter.
+a parameter and it needs to return an unique id for a single request. Server will be crashed if this method throw any errors out.
```

Comment:
There is no hard requirement for uniqueness. If it was, we will be using uuids.

Reply:
I'm sorry, but I really can't think of any other use for not unique id.🤔

Reply:
> I'm sorry, but I really can't think of any other use for not unique id.🤔

The user can choose to generate unique id but it is not the hard requirement.